packages/markdown/src/MarkdownEditor.tsx

```diff
@@ -51,6 +44,7 @@ export function MarkdownEditor(
   const [currentValue, setCurrentValue] = React.useState<string>(props.initialValue ?? '');
   const [selectedTab, setSelectedTab] = React.useState<MarkdownTab>('editor');
   const [editor, setEditor] = React.useState<InitializedEditorType | null>(null);
+  const [canUploadAssets, canUploadAssetsSet] = React.useState<boolean>(false);
```

Comment:
I was very confused when I read `canUploadAssetsSet`. This should be named `setCanUploadAssets` as the relevant verb is `set`, not `can` in this case "set whether we can upload an asset".